### ISSUE TITLE
Cant create board

### ISSUE THREAD

**Issue by Tealk** [nextcloud/deck#1032]
### Steps to reproduce
Go to Deck

### Expected behaviour
I would like to create a new board

### Actual behaviour
I don't have a button to create a new board and under settings I get the message "Limiting Deck prevents users who are not part of these groups from creating their own boards. Users can continue to work on boards that have been shared with them."

## Server configuration detail

**Operating system:** Linux 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1 (2019-04-12) x86_64

**Webserver:** Apache (fpm-fcgi)

**Database:** mysql 10.1.38

**PHP version:** 

7.3.5-1+0~20190503093827.38+stretch~1.gbp60a41b
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, cgi-fcgi, mysqlnd, PDO, xml, apcu, bcmath, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, geoip, gettext, gmp, iconv, igbinary, imagick, imap, intl, json, exif, msgpack, mysqli, pdo_mysql, apc, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Phar, memcached, redis, Zend OPcache

**Nextcloud version:** 16.0.0 - 16.0.0.9

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.2.0
 - activity: 2.9.1
 - admin_audit: 1.6.0
 - apporder: 0.7.1
 - calendar: 1.7.0
 - circles: 0.17.2
 - cloud_federation_api: 0.2.0
 - comments: 1.6.0
 - contacts: 3.1.1
 - dav: 1.9.2
 - deck: 0.6.1
 - extract: 1.2.0
 - federatedfilesharing: 1.6.0
 - federation: 1.6.0
 - files: 1.11.0
 - files_downloadactivity: 1.5.0
 - files_mindmap: 0.0.13
 - files_pdfviewer: 1.5.0
 - files_rightclick: 0.13.0
 - files_sharing: 1.8.0
 - files_texteditor: 2.8.0
 - files_trashbin: 1.6.0
 - files_versions: 1.9.0
 - files_videoplayer: 1.5.0
 - firstrunwizard: 2.5.0
 - gallery: 18.3.0
 - groupfolders: 3.0.0
 - issuetemplate: 0.5.0
 - logreader: 2.1.0
 - lookup_server_connector: 1.4.0
 - metadata: 0.9.0
 - nextcloud_announcements: 1.5.0
 - notifications: 2.4.1
 - oauth2: 1.4.2
 - password_policy: 1.6.0
 - privacy: 1.0.0
 - provisioning_api: 1.6.0
 - quota_warning: 1.5.0
 - ransomware_protection: 1.4.0
 - recommendations: 0.4.0
 - richdocuments: 3.2.4
 - serverinfo: 1.6.0
 - sharebymail: 1.6.0
 - spreed: 6.0.0
 - support: 1.0.0
 - survey_client: 1.4.0
 - systemtags: 1.6.0
 - theming: 1.7.0
 - twofactor_backupcodes: 1.5.0
 - twofactor_nextcloud_notification: 1.1.1
 - twofactor_totp: 2.1.2
 - twofactor_yubikey: 0.4.1
 - updatenotification: 1.6.0
 - viewer: 1.0.0
 - workflowengine: 1.6.0
Disabled:
 - encryption
 - files_external
 - social
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "localhost",
        "anzah.cloud",
        "mastodon.anzah.network"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "http:\/\/anzah.cloud",
    "htaccess.RewriteBase": "\/",
    "dbtype": "mysql",
    "version": "16.0.0.9",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "mysql.utf8mb4": true,
    "installed": true,
    "maintenance": false,
    "default_language": "de",
    "allow_user_to_change_display_name": true,
    "knowledgebaseenabled": false,
    "theme": "",
    "skeletondirectory": "\/var\/customers\/webs\/Anzah\/anzah.cloud\/core\/skeleton\/neu",
    "trashbin_retention_obligation": "auto",
    "versions_retention_obligation": "auto",
    "appcodechecker": true,
    "updatechecker": true,
    "updater.release.channel": "stable",
    "check_for_working_webdav": true,
    "logdateformat": "d.m.Y H:i:s",
    "logtimezone": "Europe\/Berlin",
    "log_rotate_size": 104857600,
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "PLAIN",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "ssl",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "465",
    "mail_smtpauth": 1,
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "loglevel": 2,
    "enable_previews": true,
    "preview_max_x": 2048,
    "preview_max_y": 2048,
    "preview_max_filesize_image": 50,
    "preview_max_scale_factor": 10,
    "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
    "preview_office_cl_parameters": " --headless --nologo --nofirststartwizard --invisible --norestore -convert-to pdf -outdir ",
    "enabledPreviewProviders": [
        "OC\\Preview\\PNG",
        "OC\\Preview\\JPEG",
        "OC\\Preview\\GIF",
        "OC\\Preview\\BMP",
        "OC\\Preview\\MP3",
        "OC\\Preview\\TXT",
        "OC\\Preview\\MarkDown",
        "OC\\Preview\\PDF"
    ],
    "data-fingerprint": "01b9630d063ba580c331de332a0ab360",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "filelocking.enabled": "true",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379,
        "dbindex": 10,
        "password": "***REMOVED SENSITIVE VALUE***",
        "timeout": 1.5
    },
    "twofactor_enforced": "false",
    "twofactor_enforced_groups": [
        "admin"
    ],
    "twofactor_enforced_excluded_groups": [],
    "app_install_overwrite": [
        "twofactor_yubikey"
    ]
}
```
</details>

**Are you using external storage, if yes which one:** 

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** 

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0

**Operating system:** Win10

## Logs

<details><summary>Web server error log</summary>

```
cant find anything
```
</details>

<details><summary>Nextcloud log</summary>

```
{"reqId":"CGqy8l5J3S3kfdYZWLm6","level":3,"time":"06.05.2019 12:36:36","remoteAddr":"212.211.192.4","user":"Tealk","app":"index","method":"GET","url":"\/apps\/deck\/config","message":{"Exception":"Error","Message":"Call to a member function getGID() on null","Code":0,"Trace":[{"function":"OCA\\Deck\\Controller\\{closure}","class":"OCA\\Deck\\Controller\\ConfigController","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"WEBPATH\/apps\/deck\/lib\/Controller\/ConfigController.php","line":103,"function":"array_map","args":[{"__class__":"Closure"},["*** sensitive parameter replaced ***"]]},{"file":"WEBPATH\/apps\/deck\/lib\/Controller\/ConfigController.php","line":62,"function":"getGroupLimit","class":"OCA\\Deck\\Controller\\ConfigController","type":"->","args":[]},{"file":"WEBPATH\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"get","class":"OCA\\Deck\\Controller\\ConfigController","type":"->","args":[]},{"file":"WEBPATH\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Deck\\Controller\\ConfigController"},"get"]},{"file":"WEBPATH\/lib\/private\/AppFramework\/App.php","line":126,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Deck\\Controller\\ConfigController"},"get"]},{"file":"WEBPATH\/lib\/private\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Deck\\Controller\\ConfigController","get",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"_route":"deck.Config.get"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"_route":"deck.Config.get"}]},{"file":"WEBPATH\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"_route":"deck.Config.get"}]},{"file":"WEBPATH\/lib\/base.php","line":975,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/apps\/deck\/config"]},{"file":"WEBPATH\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"WEBPATH\/apps\/deck\/lib\/Controller\/ConfigController.php","Line":107,"CustomMessage":"--"},"userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko\/20100101 Firefox\/67.0","version":"16.0.0.9"}

```
</details>

<details><summary>Browser log</summary>

cant find anything
</details>

**Comment by juliushaertl** [comment]
Can you check the output of `occ config:app:get deck groupLimit`? Does it contain a group that is not present anymore?

**Comment by Tealk** [comment]
yes there is only one group and this does not exist anymore

**Comment by juliushaertl** [comment]
A fix for this is in #1033
